IdentityCore/src/requests/sdk/adal/MSIDLegacySilentTokenRequest.m

```diff
@@ -96,7 +96,7 @@ - (BOOL)updateFamilyIdCacheWithServerError:(__unused NSError *)serverError
 - (BOOL)shouldRemoveRefreshToken:(NSError *)serverError
 {
     // ADAL removes RTs on invalid_grant
-    MSIDErrorCode oauthError = MSIDErrorCodeForOAuthError(serverError.msidOauthError, MSIDErrorInternal);
+    MSIDErrorCode oauthError = MSIDErrorCodeForOAuthErrorWithSubErrorCode(serverError.msidOauthError, MSIDErrorInternal, serverError.msidSubError);
```

Comment:
Do you need this behavior in silent token requests? I thought account transfer is always interactive.